README.md

```diff
@@ -1,33 +1,37 @@
+# Microsoft Fabric workload development kit sample
+## Creating and running custom workloads on Fabric
 
-As the maintainer of this project, please make a few updates:
+### Welcome
+Welcome to the Fabric workload development kit. This repository contains everything a Microsoft Fabric partner needs to develop and build custom workloads in Fabric, use this kit to create workloads according to your specific business needs and scenarios, and run them on Fabric with high performance, scalability, and security.
+The Fabric workload development kit is currently in public preview and is subject to change. As such, there may be issues or missing documentation. We kindly request you to keep this in mind while using the kit. We are working hard to ensure that the kit is as stable and reliable as possible, and we appreciate your patience and understanding during this time.
+If you need any further assistance, please don't hesitate to contact us at ILTriExt@microsoft.com.
 
-- Improving this README.MD file to provide a great experience
-- Updating SUPPORT.MD with content about this project's support experience
-- Understanding the security reporting process in SECURITY.MD
-- Remove this section from the README
-
-## Contributing
-
-This project welcomes contributions and suggestions.  Most contributions require you to agree to a
-Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us
-the rights to use your contribution. For details, visit https://cla.opensource.microsoft.com.
-
-When you submit a pull request, a CLA bot will automatically determine whether you need to provide
-a CLA and decorate the PR appropriately (e.g., status check, comment). Simply follow the instructions
-provided by the bot. You will only need to do this once across all repos using our CLA.
-
-This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/).
-For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or
-contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.
-
-## Trademarks
+### Trademarks
 
 This project may contain trademarks or logos for projects, products, or services. Authorized use of Microsoft 
 trademarks or logos is subject to and must follow 
 [Microsoft's Trademark & Brand Guidelines](https://www.microsoft.com/en-us/legal/intellectualproperty/trademarks/usage/general).
 Use of Microsoft trademarks or logos in modified versions of this project must not cause confusion or imply Microsoft sponsorship.
 Any use of third-party trademarks or logos are subject to those third-party's policies.
+
+### What's in the kit
+The kit contains the following components:
+- A sample workload that demonstrates how to create and run a Fabric workload using Node.js, npm, Visual Studio 2022, and the Fabric Workload DevGateway.
```

Comment:
replace with: 
A sample workload that demonstrates how to create and run a Fabric workload backend and frontend. Let's not mention technologies - in accurate.